src/components/Banners/DonateBanner.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 import "../../styles/DonateBanner.css";
-import donateImg from "../../assets/pictures/donate-banner.png";
+import donateImg from "../../assets/pictures/donate_image1.svg";
```

Comment:
Instead of `donate_image1` I think we can use relevant names. Like `Donate_banner_image` or `Donate_thumbnail`

Reply:
Ok I have changed it to donate-banner-image